docs/flink-getting-started.md

```diff
@@ -344,11 +324,11 @@ CREATE TABLE `hive_catalog`.`default`.`sample` (
 );
 ```
 
-Table create commands support the most commonly used [flink create clauses](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/sql/create.html#create-table) now, including: 
+Table create commands support the commonly used [Flink create clauses](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/table/sql/create/) including: 
```

Comment:
what about using the doc link from master branch? then we don't need to worry about the Flink versions in the doc
https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/

Reply:
Good one, the url seems to be stable